forge-gui/res/cardsfolder/e/eaten_by_spiders.txt

```diff
@@ -1,7 +1,6 @@
 Name:Eaten by Spiders
 ManaCost:2 G
 Types:Instant
-A:SP$ Destroy | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | RememberAttached$ True | SubAbility$ ChompEquip | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
-SVar:ChompEquip:DB$ DestroyAll | ValidCards$ Remembered.Equipment | SpellDescription$ Destroy all Equipment attached to that creature. | SubAbility$ Cleanup
-SVar:Cleanup:DB$ Cleanup | ClearRemembered$ True
+A:SP$ Pump | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | IsCurse$ True | SubAbility$ ChompEquip | StackDescription$ REP target_{c:Targeted} | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
```

Comment:
I think `REP target creature with flying_{c:Targeted}`

Sorry I didn't notice that before

Reply:
nah that's on me for not checking again :)